config.testnet

```diff
@@ -9,3 +9,6 @@ export INDEXER_URL=""
 export INDEXER_BRANCH=""
 export INDEXER_SHA=""
 export INDEXER_DISABLED="true"
+export ALGOSEARCH_URL="https://github.com/kevguy/algosearch"
```

Comment:
Does this tool work if no indexer is available? Perhaps leave the URL/Branch blank to disable algosearch as well.

Reply:
Just made a commit to disable the AlgoSearch if the indexer is not available in the sandbox. I was basically following what was done for indexer and made a mock NodeJS server serving status 400 if no `INDEXER_DISABLED` is `true`.